### DISCUSSION TITLE
What do researchers normally do when downloading "full anthology with abstracts"?

### DISCUSSION THREAD

**Discussion by oaro0** [acl-org/acl-anthology#2388] [Q&A]
Hello,

I have been struggling with downloading your bib file and opening it in Mendeley, Zotero and Jabref. It does not recognize the different references in the file but when I make this file smaller it does recognize the different references. It therefore seems that the file is just too big, but compressing it takes time. How should I normally approach this? Do you know why Mendeley is not able to read this gzip file?

Best,
Antonia

**Comment by akoehn** [comment]
(we have discussions?)

The problem is probably the number of entries, not the actual file size. A lot of programs cannot handle very large bibliographies and I'm that case the only thing you can do is to add papers on a case by case basis (from the publication page in the anthology).  I do that as well because I think it is good to structure one's bibliographic database (tags, notes etc) and just bulk importing skips that step.
